tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -381,7 +381,6 @@
   "DD_TRACE_WCF_RESOURCE_OBFUSCATION_ENABLED": "trace_wcf_obfuscation_enabled",
   "DD_TRACE_HEADER_TAG_NORMALIZATION_FIX_ENABLED": "trace_header_tag_normalization_fix_enabled",
   "DD_TRACE_OTEL_ENABLED": "trace_otel_enabled",
-  "DD_TRACE_OTEL_LEGACY_OPERATION_NAME_ENABLED": "trace_otel_legacy_operation_name_enabled",
```

Comment:
We shouldn't ever _remove_ lines from config_norm_rules, as older versions will still send this config value

Reply:
Ahh yeah thanks for catching, reverted